pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: collection
-version: 1.17.2
+version: 1.18.0
 description: Collections and utilities functions and classes related to collections.
 repository: https://github.com/dart-lang/collection
 
 environment:
-  sdk: ">=2.18.0 <4.0.0"
+  sdk: ">=3.0.0 <4.0.0"
```

Comment:
Or just `^3.0.0`?

Reply:
It might be cargo-cult at this point, but at some point there were some tools which couldn't handle `^...` notation in SDK versions.
Are we sure those tools are gone?

Reply:
Yes, the server filters these for SDK versions that can't read them. It should be safe to use caret constraints for packages that only support Dart 3

Reply:
@lrhn – it was like Dart 1.8.

Reply:
ACK, good to know!